0_global_functions.R

```diff
@@ -280,6 +280,6 @@ write_log <- function(msg, ...) {
     as.character(msg),
     ...
   )
-  write(composed, file = paste0(project_location, "/00_Log_Files/error_messages.txt"), append = TRUE)
+  write(composed, file = file.path(log_path,"error_messages.txt"), append = TRUE)
```

Comment:
I really don't like this stuff where we rely on some variable being available in some parent environment. Seems easy enough to pass `log_path` into the function if that's required. Otherwise, how do I know here that it's reasonable to expect that `log_path` will be available?

But, I realize this is what has been done all along, and the change you've made is low-risk because it follows the same strategy and is therefore rather easy to assess.

Reply:
You're right though and it's an easy fix. Will open a follow up issue and get on it right away
https://github.com/2DegreesInvesting/PACTA_analysis/issues/286